### PR TITLE
fix windows weirdness

### PR DESCRIPTION
**Pull request by alwaysintreble**

### CODE REVIEW

File: ap-roller.py [hunk 1]
```diff
@@ -72,15 +72,15 @@ def _b64hash(*args: Any) -> str:
 
 
 def _enter_venv(install: PathLike, venv: Optional[str] = None, msg: Optional[str] = None) -> str:
-    script = f'cd "{install}"; '
+    script = f'cd "{install}"'
```
Comment (black-sliver) [inline ap-roller.py]:
```suggestion
    script = f'cd "{install}" &&'
```

File: ap-roller.py [hunk 2]
```diff
@@ -72,15 +72,15 @@ def _b64hash(*args: Any) -> str:
...
     if is_windows and venv:
-        script += f'{venv}\\Scripts\\activate.bat; '
-        script += f'echo "{msg}"; echo "venv: %VIRTUAL_ENV%"; echo; '
+        script += f'&& {venv}\\Scripts\\activate.bat&& '
+        script += f'echo "{msg}"&& echo "venv: %VIRTUAL_ENV%"&& echo&& '
```
Comment (black-sliver) [inline ap-roller.py]:
```suggestion
        script += f'{venv}\\Scripts\\activate.bat && '
        script += f'echo "{msg}" && echo "venv: %VIRTUAL_ENV%" && echo && '
```

File: ap-roller.py [hunk 3]
```diff
@@ -72,15 +72,15 @@ def _b64hash(*args: Any) -> str:
...
     elif venv:
-        script += f'source {venv}/bin/activate; '
-        script += f'echo "{msg}"; echo "venv: $VIRTUAL_ENV"; echo; '
+        script += f'; source {venv}/bin/activate; '
+        script += f'; echo "{msg}"; echo "venv: $VIRTUAL_ENV"; echo; '
```
Comment (black-sliver) [inline ap-roller.py]:
```suggestion
        script += f'source {venv}/bin/activate && '
        script += f'echo "{msg}"; echo "venv: $VIRTUAL_ENV" && echo && '
```

File: ap-roller.py [hunk 4]
```diff
@@ -72,15 +72,15 @@ def _b64hash(*args: Any) -> str:
...
     else:
-        script += f'echo "{msg}"; echo "w/o venv"; echo; '
+        script += f'; echo "{msg}"; echo "w/o venv"; echo; '
```
Comment (black-sliver) [inline ap-roller.py]:
```suggestion
        script += f'echo "{msg}" && echo "w/o venv" && echo && '
```

**Review by black-sliver — COMMENTED** [review]
we can use && on non-windows as well

File: ap-roller.py (hunk 1 above)
Comment (alwaysintreble) [reply ap-roller.py]: no

**Comment by black-sliver** [comment]
Thanks!

**Pull request merged**